Makefile

```diff
@@ -1,123 +1,110 @@
+#   * It must at least contains one value.
+#   * Values can be the app ticker or anything else but should be unique.
+VARIANT_PARAM = COIN
+VARIANT_VALUES = sia
+
+# Enabling DEBUG flag will enable PRINTF and disable optimizations
+#DEBUG = 1
+
+########################################
+#     Application custom permissions   #
+########################################
+# See SDK `include/appflags.h` for the purpose of each permission
+#HAVE_APPLICATION_FLAG_DERIVE_MASTER = 1
+#HAVE_APPLICATION_FLAG_GLOBAL_PIN = 1
+#HAVE_APPLICATION_FLAG_BOLOS_SETTINGS = 1
+#HAVE_APPLICATION_FLAG_LIBRARY = 1
+
+########################################
+# Application communication interfaces #
+########################################
+ENABLE_BLUETOOTH = 1
+#ENABLE_NFC = 1
+
+########################################
+#         NBGL custom features         #
+########################################
+#ENABLE_NBGL_QRCODE = 1
+#ENABLE_NBGL_KEYBOARD = 1
+#ENABLE_NBGL_KEYPAD = 1
+
+########################################
+#          Features disablers          #
+########################################
+# These advanced settings allow to disable some feature that are by
+# default enabled in the SDK `Makefile.standard_app`.
+#DISABLE_STANDARD_APP_FILES = 1 
+#DISABLE_DEFAULT_IO_SEPROXY_BUFFER_SIZE = 1 # To allow custom size declaration
+#DISABLE_STANDARD_APP_DEFINES = 1 # Will set all the following disablers
+#DISABLE_STANDARD_SNPRINTF = 1
+#DISABLE_STANDARD_USB = 1
+#DISABLE_STANDARD_WEBUSB = 1
+#DISABLE_STANDARD_BAGL_UX_FLOW = 1
 DEFINES += HAVE_LEGACY_PID
-DEFINES += APPNAME=\"$(APPNAME)\"
-DEFINES += APPVERSION=\"$(APPVERSION)\"
-DEFINES += UNUSED\(x\)=\(void\)x
-
-ifeq ($(TARGET_NAME),$(filter $(TARGET_NAME),TARGET_NANOX TARGET_NANOS))
-DEFINES += HAVE_BAGL
-DEFINES += HAVE_UX_FLOW
-endif
-
-### Nano X
-ifeq ($(TARGET_NAME),TARGET_NANOS)
-DEFINES += IO_SEPROXYHAL_BUFFER_SIZE_B=128
-else ifeq ($(TARGET_NAME),$(filter $(TARGET_NAME),TARGET_NANOX TARGET_STAX))
-DEFINES += IO_SEPROXYHAL_BUFFER_SIZE_B=300
-# bluetooth
-DEFINES += HAVE_BLE BLE_COMMAND_TIMEOUT_MS=2000
-DEFINES += HAVE_BLE_APDU
-# include fonts or ui will be empty
-SDK_SOURCE_PATH += lib_blewbxx lib_blewbxx_impl
-endif
-
-ifeq ($(TARGET_NAME),TARGET_NANOX)
-DEFINES += HAVE_BAGL_ELLIPSIS
-DEFINES += BAGL_WIDTH=128 BAGL_HEIGHT=64
-DEFINES += HAVE_BAGL_FONT_OPEN_SANS_REGULAR_11PX
-DEFINES += HAVE_BAGL_FONT_OPEN_SANS_EXTRABOLD_11PX
-DEFINES += HAVE_BAGL_FONT_OPEN_SANS_LIGHT_16PX
-endif
- 
-
-##############
-#  Compiler  #
-##############
-
-CC := $(CLANGPATH)clang
-CFLAGS += -O3 -Os
-
-AS := $(GCCPATH)arm-none-eabi-gcc
-LD := $(GCCPATH)arm-none-eabi-gcc
-LDFLAGS += -O3 -Os
-LDLIBS += -lm -lgcc -lc 
-
-##################
-#  Dependencies  #
-##################
-
-# import rules to compile glyphs
-include $(BOLOS_SDK)/Makefile.glyphs
-# import generic rules from the sdk
-include $(BOLOS_SDK)/Makefile.rules
-
-dep/%.d: %.c Makefile
+DEFINES += HAVE_LEDGER_ASSERT_DISPLAY LEDGER_ASSERT_CONFIG_MESSAGE_INFO
```

Comment:
This is not recommended for release builds.
When building with DEBUG=1, everything is already enable, and when building without (for production deployment) I'm not sure you want such debug screens to be displayed on the user device (if the assert where to fail)? Also, this will increase the app flash size not really necessarily?